aten/src/ATen/native/mps/OperationUtils.mm

```diff
@@ -194,7 +194,7 @@ MPSDataType getMPSScalarType(ScalarType scalar_type) {
         str += getMPSTypeString(tensor.scalar_type()) + "[";
         // if tensor is a scalar
         if (tensor.dim() == 0) {
-          str += std::to_string(getMPSScalarValue(tensor));
+          str += (use_scalar_value ? std::to_string(getMPSScalarValue(tensor)) : "Scalar");
```

Comment:
Here if the tensor is a scalar, what's the condition we won't need to use the Scalar value.